<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OldFileItemTemplates</class>
 <widget class="QWidget" name="OldFileItemTemplates">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>49</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>OldFileItemTemplates</string>
  </property>
  <widget class="QLabel" name="label_fileName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>文件名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_filePath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>25</y>
     <width>240</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:grey;</string>
   </property>
   <property name="text">
    <string>全路径</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>5</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>不拷贝，仅在目的地址创建链接文件指向该文件</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #222;</string>
   </property>
   <property name="text">
    <string>创建链接</string>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
